src/libs/antares/study/parts/short-term-storage/properties.cpp

```diff
@@ -21,6 +21,7 @@
 
 #include <antares/logs/logs.h>
 #include <stdexcept>
+#include <boost/algorithm/string.hpp>
```

Comment:
```suggestion
#include <boost/algorithm/string/case_conv.hpp>
```
<[boost/algorithm/string/case_conv.hpp](https://www.boost.org/doc/libs/1_84_0/boost/algorithm/string/case_conv.hpp)>

Reply:
Done